src/backend/app/projects/project_crud.py

```diff
@@ -26,41 +65,37 @@ async def get_project_by_id(
     )
     return await convert_to_app_project(db_project)
 
-
-async def convert_to_app_project(db_project: db_models.DbProject):
-    """Legacy function to convert db models --> Pydantic.
-
-    TODO refactor to use Pydantic model methods instead.
-    """
-    if not db_project:
-        log.debug("convert_to_app_project called, but no project provided")
-        return None
-
-    app_project = db_project
-
-    if db_project.outline:
-        app_project.outline_geojson = geometry_to_geojson(
-            db_project.outline, {"id": db_project.id}, db_project.id
-        )
-    app_project.tasks = db_project.tasks
-    return app_project
-
-
 async def get_projects(
-    db: Session,
+    db:Database,
```

Comment:
Is this type correct?

It seems like it's probably a SessionLocal object, but I could be wrong